packages/t/timeshift/package.yml

```diff
@@ -0,0 +1,28 @@
+name       : Timeshift
+version    : 24.01.1
+release    : 1
+source     :
+    - https://github.com/linuxmint/timeshift/archive/refs/tags/master.mint21.tar.gz : 7272f5007de6e831e55bbae8ffe1b5bd176c304c4a86f65ef62fb6cb08b556f3
+homepage   : https://github.com/linuxmint/timeshift/tree/master.mint21
+license    : GPL-2.0-or-later # CHECK ME
+component  : system.utils
+summary    : Backup and restore tool for BTRFS and RSYNC.
+description: |
+    Timeshift for Linux is an application that provides functionality similar to the System Restore feature in Windows and the Time Machine tool in Mac OS. Timeshift protects your system by taking incremental snapshots of the file system at regular intervals. These snapshots can be restored at a later date to undo all changes to the system.
+builddeps  :
```

Comment:
See the warning about the correct order.
Also, we prefer to use the pkgconfig() version of dependencies when they are available. For instance: `eopkg ingo libgee-devel` shows that the package provides `pkgconfig(gee-0.8)`, so you would replace `libgee-devel` with that

Reply:
Updated all of them.